From baf1eeb398d0201260a11a9ba9270461da7a0dc3 Mon Sep 17 00:00:00 2001
From: allgdante <allan.garret@gmail.com>
Date: Wed, 4 Jul 2018 17:08:11 +0200
Subject: [PATCH 09/11] parser: remove specific tests for ofile

musl libc doesn't implement this feature, so the test fail.
We must find a way to skip this test instead of removing it
---
 parser/tst/simple_tests/rlimits/ok_rlimit_09.sd | 7 -------
 1 file changed, 7 deletions(-)
 delete mode 100644 parser/tst/simple_tests/rlimits/ok_rlimit_09.sd

diff --git a/parser/tst/simple_tests/rlimits/ok_rlimit_09.sd b/parser/tst/simple_tests/rlimits/ok_rlimit_09.sd
deleted file mode 100644
index 6510ae72..00000000
--- a/parser/tst/simple_tests/rlimits/ok_rlimit_09.sd
+++ /dev/null
@@ -1,7 +0,0 @@
-#
-#=DESCRIPTION simple max open file (same as nofile) rlimit test
-#=EXRESULT PASS
-
-profile rlimit {
-  set rlimit ofile <= 1234,
-}
-- 
2.17.1

